app/src/pages/game.tsx

```diff
@@ -50,7 +53,7 @@ function FreePlay() {
           backgroundColor: '#B12929',
           width:'9rem', height:'3rem', fontSize:17, borderRadius: 5,
         }}
-        onClick={cancelGame}
+        onClick={function(){return setCancel(true)}}
```

Comment:
musst du in dem Fall nicht als function declaren. entweder direct aufrufen mit `() => {setCancel(true)}` oder du deklarierst irgendwo einen const cast auf das gleiche, also `const openCancelGameModal = () => {setCancel(true)}` und nutzt dann openCancelGameModal